code/game/objects/items/storage/boxes.dm

```diff
@@ -609,6 +609,39 @@
 	playsound(loc, "rustle", 50, 1, -5)
 	user.visible_message("<span class='notice'>[user] hugs \the [src].</span>","<span class='notice'>You hug \the [src].</span>")
 
+/////clown box & honkbot assembly
+obj/item/storage/box/clown
+	name = "clown box"
+	desc = "A colorful cardboard box for the clown"
+	icon_state = "clownbox"
+	illustration = null
+	//foldable = null
+
+/obj/item/stack/sheet/cardboard/attackby(obj/item/stamp/clown/I, mob/user, params)
+	if(!istype(I, /obj/item/stamp/clown))

```

Comment:
It would make more sense to define if it *is* attacked by something, have the code there, then return, than to do this, in my opinion. That or `else return ..()` it.